osgi-dependencies/proxy/src/main/java/com/cognifide/aet/proxy/ProxyServerProvider.java

```diff
@@ -44,19 +44,20 @@
           unbind = "unbindProxyManager")
   private final Map<String, ProxyManager> collectorManagers = Maps.newConcurrentMap();
 
-  public ProxyServerWrapper createProxy(String useProxy, int port) throws ProxyException {
-    String proxyType = useProxy;
-    if ("true".equals(useProxy)) {
+  public ProxyServerWrapper createProxy(String usedProxy) throws ProxyException {
+    String proxyType = usedProxy;
+    if ("true".equals(usedProxy)) {
```

Comment:
String proxyType = usedProxy;
    if ("true".equals(usedProxy)) {
   //...
   proxyType = DEFAULT_PROXY_MANAGER;

The values that you assign to the `proxyType` variable here are `"true"` and `"embedded"`... That's stringly typed and difficult to follow.

Did you actually mean to use the same variable? If so, consider using an explicit `enum` type to cover the possible values. If not, use separate variables for these values and use `boolean` when actually handling boolean variables.